src/components/panel/panel.js

```diff
@@ -338,14 +338,13 @@ angular
  * Overlapping the panel with an element:
  * `new MdPanelPosition()
  *     .relativeTo(someElement)
- *     .withPanelXPosition($mdPanel.xPosition.ALIGN_START)
- *     .withPanelYPosition($mdPanel.yPosition.ALIGN_TOPS);`
+ *     .withPanelPosition($mdPanel.xPosition.ALIGN_START,
+ *        $mdPanel.yPosition.ALIGN_TOPS);`
  *
  * Aligning the panel with the bottom of an element:
  * `new MdPanelPosition()
  *     .relativeTo(someElement)
- *     .withPanelXPosition($mdPanel.xPosition.CENTER)
- *     .withPanelYPosition($mdPanel.yPosition.BELOW);`
+ *     .withPanelPosition($mdPanel.xPosition.CENTER, $mdPanel.yPosition.BELOW);
```

Comment:
Please format this the same as the above example.

Also add an example here that demonstrates the tryPanelPositions function.